client/my-sites/upgrades/index.js

```diff
@@ -193,7 +193,7 @@ module.exports = function() {
 			upgradesController.googleAppsWithRegistration
 		);
 
-		page( '/domains/add/mapping/:domain',
+		page( '/domains/add/mapping/:initial_query?/:domain',
```

Comment:
Thanks for the info, @stephanethomas, for some reason I thought it was snake_case. I'll keep it in mind in the future :+1: